e2e-tests/tests/translation-version-name-appears-in-title.spec.ts

```diff
@@ -0,0 +1,80 @@
+        sourceLanguage: 'Hebrew', sourceLanguageToggle: SOURCE_LANGUAGES.HE,
+        translations: 'תרגומים', select: 'בחירה', currentlySelected: 'נוכחי'}, 
+
+    // Hebrew Interface and Bilingual Source
+    {interfaceLanguage: 'Hebrew', interfaceLanguageToggle: LANGUAGES.HE, 
+        sourceLanguage: 'Bilingual', sourceLanguageToggle: SOURCE_LANGUAGES.BI,
+        translations: 'תרגומים', select: 'בחירה', currentlySelected: 'נוכחי'}, 
+    
+    // Hebrew Interface and English Source
+    {interfaceLanguage: 'Hebrew', interfaceLanguageToggle: LANGUAGES.HE, 
+        sourceLanguage: 'English', sourceLanguageToggle: SOURCE_LANGUAGES.EN,
+        translations: 'תרגומים', select: 'בחירה', currentlySelected: 'נוכחי'}, 
+
+    // English Interface and English Source
+    {interfaceLanguage: 'English', interfaceLanguageToggle: LANGUAGES.EN, 
+        sourceLanguage: 'English', sourceLanguageToggle: SOURCE_LANGUAGES.EN,
+        translations: 'Translations', select: 'Select', currentlySelected: 'Currently Selected'},
+
+    // English Interface and English Source
+    {interfaceLanguage: 'English', interfaceLanguageToggle: LANGUAGES.EN, 
+        sourceLanguage: 'Bilingual', sourceLanguageToggle: SOURCE_LANGUAGES.BI,
+        translations: 'Translations', select: 'Select', currentlySelected: 'Currently Selected'},
+    
+    // English Interface and Hebrew Source
+    {interfaceLanguage: 'English', interfaceLanguageToggle: LANGUAGES.EN, 
+        sourceLanguage: 'Hebrew', sourceLanguageToggle: SOURCE_LANGUAGES.HE,
+        translations: 'Translations', select: 'Select', currentlySelected: 'Currently Selected'}
+
+].forEach(({interfaceLanguage, interfaceLanguageToggle, sourceLanguage, sourceLanguageToggle, translations, currentlySelected, select}) => {
+    test(`${interfaceLanguage} - translation name appears in title for ${sourceLanguage} source text`, async ({ context }) => {
+         // Navigate to Bereshit in specified Interface Language
+         const page = await goToPageWithLang(context,'/Genesis.1', `${interfaceLanguageToggle}`)
+
+         // Change the Source Language of the text
+         await changeLanguageOfText(page, `${sourceLanguageToggle}`)
+ 
+         // Retain the translation name locator
+         const translationNameInTitle = page.locator('span.readerTextVersion')
+ 
+         // Navigate to the Translations sidebar by clicking on the text title
+         //Clicks on בראשית א׳ / Genesis I
+         await page.locator('h1').click()
+         
+         // Click on Translations
+         await page.getByRole('link', {name: `${translations}`}).click()
+         
+         // Wait for Translations side-bar to load by waiting for 'Translations' header
+         await page.waitForSelector('h3')
+
+        // Check if the default translation in the title matches the selected translation
+        // NOTE: We are skipping checking for the default translation here, due to the Hebrew text being default Masoretic
```

Comment:
Can we rewrite to say "due to the Hebrew text not displaying a version title". That might be a more accurate description of what's going on in terms of site functionality (even if the root reason behind it is what you described here, for Tanakh texts at least)

Reply:
Sure thing!  I will make this change.

Reply:
Great, thank you!